test/application/accomodation_loader/accomodation_loader_bloc_test.dart

```diff
@@ -28,7 +28,7 @@ void main() {
       // assert
       expect(
         accomodationLoaderBloc.state,
-        const AccomodationLoaderState.initial(),
+        const AccommodationLoaderState.initial(),
```

Comment:
The bloc is renamed here but there are no changes to its definition, this should be all red in the IDE/tests

Please revert changes to this file or rename the AccomodationLoader to AccommodationLoader as well